microsoft-edge/web-platform/ad-selection-api.md

```diff
@@ -0,0 +1,300 @@
+
+1. In the **Ad Selection API** section, click the **Register** button.  The **Origin Trial Registration** form opens:
+
+   ![Registration form](./ad-selection-api-images/reg-form.png)
+
+1. Fill in the form.
+
+   * In the **Domain** text box, enter a single top-level domain.  For example, a top-level domain is `contoso.example`, and optionally includes subdomains such as `subdomain.contoso.example`.
+
+   * In the **Subdomain Support** section, select the **Yes** or **No** option button.
+
+   * In the **Email Address** text box, provide a valid developer contact for the domain.
+
+   * In the **Origin Trial Name** text box, enter **Ad Selection API**.
+
+1. Click the **Submit** button.
+
+   An Origin Trial token is generated for the top-level domain and is sent to you.  After you receive the token, use the token in the `attestations.json` file for that domain, to enable testing the Ad Selection API with supported Microsoft Edge clients.
+
+1. To complete your attestation and allow continued access to the Ad Selection API, create a JSON file named `attestations.json` for the domain and make it available at the `/.well-known/attestations.json` path for that domain.  For example: `https://contoso.example/.well-known/ad-selection-attestations.json`.
+
+   The `attestations.json` JSON file must be published within **30 days** of receiving the OT token.
+
+
+<!-- ------------------------------ -->
+#### Example attestations.json file
+
+The `attestations.json` file for a domain must conform to the following standards below.  The file must include the generated OT token for the top-level domain, as well as a list of APIs (features that are associated with the Ad Services API).
+
+The following is an example `attestations.json` file:
+
+```json
+{
+    "ad_selection_api_attestations": [
+        {
+            "attestation_parser_version": "2",
+            "attestation_version": "2",
+            "privacy_policy": [
+                "https://contoso.example/privacy/"
+            ],
+            "ownership_token": "<Your Unique Token Here>",
+            "enrollment_site": "https://contoso.example/",
+            "platform_attestations": [
+                {
+                    "platform": "edge",
+                    "attestations": {
+                        "attribution_reporting_api": {
+                            "ServiceNotUsedForIdentifyingUserAcrossSites": true/false
+                        },
+                        "shared_storage_api": {
+                            "ServiceNotUsedForIdentifyingUserAcrossSites": true/false
+                        },
+                        "private_aggregation_api": {
+                            "ServiceNotUsedForIdentifyingUserAcrossSites": true/false
+                        },
+                        "ad_selection_api": {
+                            "ServiceNotUsedForIdentifyingUserAcrossSites": true/false
+                        }
```

Comment:
```suggestion
                        "ad_selection_api": {
                            "ServiceNotUsedForIdentifyingUserAcrossSites": true/false
                        },
                        "attribution_reporting_api": {
                            "ServiceNotUsedForIdentifyingUserAcrossSites": true/false
                        },
                        "shared_storage_api": {
                            "ServiceNotUsedForIdentifyingUserAcrossSites": true/false
                        },
                        "private_aggregation_api": {
                            "ServiceNotUsedForIdentifyingUserAcrossSites": true/false
                        }
```
shouldn't "private_aggregation_api" be listed first, like in flag's desc?  this is the main "api".  the others are "associated features".
![image](https://github.com/user-attachments/assets/9e9f1971-60fb-44a4-862c-cacdd4e4cf80)

Reply:
sounds good!

Reply:
reordered members